krabs/krabs/etw.hpp

```diff
@@ -305,6 +305,12 @@ namespace krabs { namespace details {
                 // StartTrace() actually sets this to 0 on failure
                 trace_.registrationHandle_ = INVALID_PROCESSTRACE_HANDLE;
             }
+            catch (invalid_parameter) {
+                (void)open_trace();
```

Comment:
could you add a comment on the line after `catch` describing the scenario that you are addressing with this change, so it's clear to future maintainers of this code?